spikeextractors/extractors/neuroscopesortingextractor/neuroscopesortingextractor.py

```diff
@@ -12,7 +12,18 @@ class NeuroscopeSortingExtractor(SortingExtractor):
     The .clu file is a file with one more row than the .res with the first row corresponding to
     the total number of unit ids and the rest of the rows indicating which unit id the corresponding
     entry in the .res file refers to.
-    Unit ID 1 is to be treated as noise.
+    
+    In the original Neuroscope format:
+        Unit ID 0 is the cluster of unsorted spikes (noise).
+        Unit ID 1 is a cluster of multi-unit spikes.
+        
+    The function defaults to returning multi-unit activity.
+    To return only the fully sorted units, set keep_mua_units=False.
+        
+    The sorting extractor always returns unit IDs from 1, ..., number of chosen clusters.
+        
+    Until get_unsorted_spike_train is implemented into the base for sortingextractor objects,
+    that group will be ignored by this function (consistent with the original implementation).
```

Comment:
@CodyCBakerPhD just to be clear the `get_unsorted_spike_train` is NOT implemented in the SortingExtractor (see https://github.com/SpikeInterface/spikeextractors/blob/master/spikeextractors/sortingextractor.py#L690-L718). It could be overrridden here, but I think it wont be a common use case :)

Reply:
Modified documentation to emphasize